src/stepper/Stepper.tsx

```diff
@@ -203,60 +185,39 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
     const buttonSize = StepperButtonSize[display]
 
     return (
-      <div className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}>
-        <input
-          tabIndex={0}
-          className="kirk-stepper-range"
-          type="range"
-          min={this.props.min}
-          max={this.props.max}
-          defaultValue={`${this.state.value}`}
-          step={this.props.step}
-          aria-valuemin={this.props.min}
-          aria-valuemax={this.props.max}
-          aria-valuenow={this.state.value}
-          onKeyDown={this.handleKeyDown}
-        />
+      <div
+        className={cc(['kirk-stepper', `kirk-stepper-${display}`, className])}
+        aria-label={children}
+      >
         <Button
-          tabIndex="-1"
+          aria-label={decreaseLabel}
           type="button"
           className="kirk-stepper-decrement"
           status={ButtonStatus.UNSTYLED}
           disabled={isMin}
           isBubble
           {...this.createButtonListeners(this.decrement)}
         >
-          <MinusIcon
-            title={decreaseLabel}
-            iconColor={isMin ? color.disabled : color.primary}
-            size={buttonSize}
-          />
+          <MinusIcon iconColor={isMin ? color.disabled : color.primary} size={buttonSize} />
```

Comment:
Does it change the screen reader output if `decreaseLabel ` is passed to the icon ?

Reply:
Not much change, only a minor one. When you have the focus on the button in browsing mode you could still go down to the icon to read the title. Now it says "No title to interact with" but it's reading the button label correctly. Also in code, I feel it makes more sense to put the label on the Button since this is element which contains the action logic.